### ISSUE TITLE
Recent changes break `DJANGO_VITE_STATIC_URL_PREFIX`

### ISSUE THREAD

**Issue by scott-8** [MrBin99/django-vite#76]
`DJANGO_VITE_STATIC_URL_PREFIX` needs to end in a trailing slash, or else `urljoin` won't work (see https://github.com/MrBin99/django-vite/issues/36). https://github.com/MrBin99/django-vite/pull/52 recently redid how URLs are generates, and bypasses the step ensuring there is a trailing `/`. An example:
```bash
>>> from urllib.parse import urljoin
>>> urljoin("static", "files")
'files'
>>> urljoin("static/", "files")
'static/files'
```
A check should be added back to ensure `urljoin` will work. See [this comment](https://github.com/MrBin99/django-vite/pull/52#issuecomment-1569876788) for another example.

**Comment by thijskramer** [comment]
@scott-8 I've created a fix (#78), could you please check if that is the solution you had in mind?

**Comment by scott-8** [comment]
Yes that should work!
